constraints/miden-vm/range_checker_8bit.air

```diff
@@ -0,0 +1,23 @@
+mod RangeChecker8BitAir
+
+### Helper functions ##############################################################################
+
+# Returns array of mutually exclusive multiplicity flags.
+# p[0] set to 1 when we don't include the value into the running product.
+# p[1] set to 1 when we include the value into the running product.
+# p[2] set to 1 when we include two copies of the value into the running product.
+# p[3] set to 1 when we include four copies of the value into the running product.
+fn get_multiplicity_flags(s0: scalar, s1: scalar) -> vector[4]:
+    return [!s0 & !s1, s0 & !s1, !s0 & s1, s0 & s1]
+
+### 8-bit Range Checker Air Constraints #################################################################
+
+ev range_checker_8bit(main: [t, s0, s1, v], aux: [p0]):
```

Comment:
Let's add a comment above this evaluator. It can reuse some of the explanation from the [8-bit range checks table](https://0xpolygonmiden.github.io/miden-vm/design/range.html#8-bit-range-checks-table) section

Reply:
Also, we can remove the `main` and `aux` keywords